packages/grafana-data/src/types/vector.ts

```diff
@@ -13,42 +13,46 @@ declare global {
 
 // JS original sin
 // this if condition is because Jest will re-exec this block multiple times (in a browser this only runs once)
-if (!Object.getOwnPropertyDescriptor(Array.prototype, 'toArray')) {
-  Object.defineProperties(Array.prototype, {
-    get: {
-      value: function (idx: number) {
-        return this[idx];
+export function patchArrayVectorProrotypeMethods() {
```

Comment:
nothing in the prototype patching code changed. only now wrapped into this function to be reusable inside the sandbox.

Reply:
if it works for you then  👍 

all this monkey patching stuff is temporary anyways, and will be removed in v11 along with all the Vector classes/interfaces.